portfolio/src/main/java/com/google/sps/servlets/CommentsDatastore.java

```diff
@@ -0,0 +1,70 @@
+package com.google.sps.servlets;
```

Comment:
FYI, you should have moved this service outside of servlet since it's not API related it's about data manipulation. Don't need to change it for this CL, just letting you know for next project.